README.md

```diff
@@ -111,23 +111,43 @@ Next / interim revision example:
 
 The generated `.vsix` will be under folder `./dist`
 
-## Publishing vscode-builtin-extensions to open-vsx
+## Packaging builtin-extensions as a builtin-extension-pack '.vsix`
 
-The `ovsx` client is used to publish to an open-vsx registry. By default https://open-vsx.org is used. To use another open-vsx registry, set its URL in environment variable `OVSX_REGISTRY_URL`. Use enviromnent variable `OVSX_PAT` to store the personal access token obtained from the registry.
+The version of the builtin extension-pack corresponds to the `version` present in the vscode submodule's `package.json`. For `next` versions, an appropriate hash suffix is added.
 
-After packaging the extensions as `.vsix` (see above), you may examine/test them under folder `dist`. Remove any that you do not wish to be published (e.g. those not working well). When ready proceed with publishing:
+Latest / solid revision example:
+
+    cd vscode; git checkout 1.45.0; cd ..
+    yarn; yarn create-extension-pack:latest
+
+Next / interim revision example:
+
+    cd vscode; git checkout d69a79b73808559a91206d73d7717ff5f798f23c; cd ..
+    yarn; yarn create-extension-pack:next
+
+The generated `.vsix` will be under the folder `./dist`
+
+## Publishing vscode-builtin-extensions and builtin-extension-packs to open-vsx
+
+The `ovsx` client is used to publish to an open-vsx registry. The default registry is set to the public instance at https://open-vsx.org.
+
+The environment variable `OVSX_REGISTRY_URL` may be set to configure publishing to a different registry URL.
+
+The environment variable `OVSX_PAT` is used to set the personal access token obtained from the registry.
+
+After packaging the extensions and extension packages as `.vsix` (see above), you may examine/test them under the `dist` folder. Remove any extension that you do not wish to be published (e.g. those not working well), and when ready proceed with publishing:
 
     yarn publish:vsix
 
-## Re-publishing vscode-builtin-extensions to open-vsx
+## Re-publishing vscode-builtin-extensions and builtin-extension-packs to open-vsx
```

Comment:
I suggest: "Re-publishing individual vscode built-in extensions and the built-in extensions pack to open-vsx"

Reply:
As per the comment above, it seems the corresponding name would be:
`Re-publishing individual built-in vscode extensions and built-in extension packs to open-vsx.`